### PR TITLE
feat: Write next version gradle

### LINKED ISSUE #81: Write next version in build.gradle
### Feature Request

Writing calculated next version back into the build.gradle file

### PR DESCRIPTION
**Pull request by philippart-s**
fix #81 

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [X] Link to relevant issues in GitHub or Jira
- [X] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

I don't know gradle command to do the stuff (like maven) so I did it with Java code.
I don't use Properties class to load and write the properties file because this class is too "java" oriented 😉 
I assume that the version is in the gradle.properties file, not sure it's the right think :wink: .

### CODE REVIEW

**Comment by philippart-s** [comment]
@ADI10HERO ok for review :wink:

File: src/test/java/io/jenkins/plugins/conventionalcommits/utils/GradleProjectTypeTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    File gradleDir = rootFolder.newFolder("SampleGradleProject");
+    createBuildGradleFiles(gradleDir, buildGradleWithoutVersionContent);
+
+    // When : write next version tp the file
+    GradleProjectType gradleProjectType = new GradleProjectType();
+    gradleProjectType.writeVersion(gradleDir, Version.valueOf("1.1.0"), null);
+
+    // Then : IOException is thrown
+  }
+}
```
Comment (ADI10HERO) [inline src/test/java/io/jenkins/plugins/conventionalcommits/utils/GradleProjectTypeTest.java:136]:
```suggestion
}

```

**Review by ADI10HERO — APPROVED** [review]
This looks good to me @philippart-s 

Thank you so much for adding those IOException tests as well :)

**Comment by philippart-s** [comment]
you're welcome 😉

**Pull request merged**